### DISCUSSION TITLE
The most convenient way to create a new file

### DISCUSSION THREAD

**Discussion by wutchzone** [helix-editor/helix#6052] [Q&A]
What is the most convenient way to create a new file **relative** to the current opened file?

I know there is the `:open <name_of_new_file>` . However, the path is relative to where I opened the helix. Is there a way to make `:open` relative to the current opened file (focused buffer)?

Thank you for any hint!

**Comment by LEI** [comment]
To open a file picker at the current buffer's directory #4666 introduced a new command, for example:
```toml
[keys.normal]
minus = "file_picker_in_current_buffer_directory"
```
It only works for existing files in the current directory or sub-directories though, command expansions are discussed in #3134.

**Reply by areux** [reply]
The question was not how to "open a file", it's how to create a new one.
You can't create a new file in file picker.

**Reply by kirawi** [reply]
You can do `:o` `<C-r>%` delete file name and enter new file name.

**Comment by Keshav-writes-code** [comment]
this is so wild!! that Helix is missing core IDE functionality and still Advertising itself as an IDE

**Reply by kirawi** [reply]
To be pedantic, Helix advertises itself as a post-modern text editor with IDE features. You are probably looking for https://github.com/helix-editor/helix/pull/12902

**Comment by franzb** [comment]
For now I'm using yazi for creating new files. Works like a charm with this setup: https://yazi-rs.github.io/docs/tips/#helix

**Comment by unsecretised** [comment]
Is there still no better way? Would be nice if the file picker could also create new files as a fallback..
